dace/codegen/targets/xilinx.py

```diff
@@ -495,10 +495,10 @@ def generate_kernel_boilerplate_pre(self, sdfg, state_id, kernel_name,
                         memory_bank[1], sdfg)
                 else:
                     lowest_bank_index = int(memory_bank[1])
-                for bank in fpga.iterate_hbm_multibank_arrays(
-                        data_name, data, sdfg):
+                for bank, if_id in fpga.iterate_hbm_interface_ids(
```

Comment:
Don't use `if` as an abbreviation, this is confusing :-) Just write out `interface_id`